book/_quarto.yml

```diff
@@ -36,6 +36,7 @@ book:
     - technical.qmd
     - interpretation.qmd
     - extending.qmd
+    - bayesian_optimization.qmd
```

Comment:
Obviously will need to think more carefully about where this goes in the long run